subsys/logging/Kconfig.processing

```diff
@@ -74,14 +74,15 @@ config LOG_PROCESS_THREAD_SLEEP_MS
 
 config LOG_PROCESS_THREAD_STACK_SIZE
 	int "Stack size for the internal log processing thread"
-	default 2048 if COVERAGE_GCOV
-	default 1024 if NO_OPTIMIZATIONS
-	default 1024 if XTENSA
 	default 4096 if (X86 && X86_64)
 	default 4096 if ARM64
 	default 4096 if SPARC
+	default 2048 if COVERAGE_GCOV
 	default 2048 if (RISCV && 64BIT)
 	default 2048 if LOG_BACKEND_FS
+	default 1152 if LOG_BACKEND_NET
```

Comment:
just a note that in CONFIG_USERSPACE builds on ARMv7-M, this gets rounded up to 2048